py3status/modules/do_not_disturb.py

```diff
@@ -90,13 +90,14 @@ class Dunst(Notification):
     """
 
     def toggle(self, state):
-        # if not in pause mode we delete all pending notifications in the queue
-        # before resuming
-        if self.parent.pause is False and state is False:
-            self.parent.py3.command_run("killall --signal SIGTERM dunst")
-        self.parent.py3.command_run(
-            "killall --signal {} dunst".format(self.parent.signals[state])
-        )
+        if state:
+            self.parent.py3.command_run("pkill -SIGUSR1 dunst")  # pause
```

Comment:
I like this refactored way, especially seeing the signals right here in the command instead of them coming from `self.parent.signals` — it wasn't obvious at all to ensure that `signals` list contained correct signals in the correct order. @lasers are you fine with having this approach merged?